translate_test.go

```diff
@@ -58,5 +73,3 @@ func TestTranslationStability(t *testing.T) {
 		}
 	}
 }
-
-// TODO: test for not modifying the original PV.
```

Comment:
nit: leave this TODO in here, we still want this test I think.

Reply:
I thought the current implementation is already not modifying the original PV? maybe I miss understand the TODO.

Reply:
i c. updated